tests/ocpp-server/server.py

```diff
@@ -127,6 +175,14 @@ async def on_connect(websocket, path):
 
     charge_point_id = path.strip("/")
     cp = ChargePoint(charge_point_id, websocket)
+
+     # Check if request argument is specified
+    if args.request:
+        asyncio.create_task(send_ocpp_command(cp, args.request, args.delay, args.period))
```

Comment:
It's not at connection opening that command sending is expected to start: 

- A global variable is used instead of library callback arguments  => design error
- A reference to the asyncio is used in an object used be the event loop: the two are orthogonal => design issue

CLI args parsing must be done in the main function. And the output used here to iter the ChargingPoints set() of connected charging stations to run an asyncio task.